file1.txt

```diff
@@ -1,3 +1,3 @@
 Wstęp
 
-Ta książka poświęcona jest  współczesnym problemom ekonomicznym świata 
+Ta książka poświęcona jest  współczesnym problemom ekonomicznym świata. 
```

Comment:
Marcy , sentence with dot at the end of sentence is ok for me.

Reply:
I've merged with master on remote by CLI in my local rep but nothing is changed. Do you know Why

Reply:
Ok I am going to merge and let's see